sycl/doc/extensions/experimental/sycl_ext_intel_fpga_task_sequence.asciidoc

```diff
@@ -0,0 +1,500 @@
+  using value_t = property_value<balanced_key>;
+};
+
+struct invocation_capacity_key {
+  template <uint32_t Size>
+  using value_t = property_value<invocation_capacity_key,
+    std::integral_constant<uint32_t, Size>>;
+};
+
+struct response_capacity_key {
+  template <uint32_t Size>
+  using value_t = property_value<response_capacity_key,
+    std::integral_constant<uint32_t, Size>>;
+};
+
+inline constexpr balanced_key::value_t balanced;
+
+template <> struct is_property_key<balanced_key> : std::true_type {};
+
+// TODO: Not sure if the template here should just be
+// template<auto &f, class propertiesT>
+template <typename ReturnT, typename ... ArgsT, ReturnT(&f) (ArgsT...),
+  class propertiesT>
+struct is_property_key_of<balanced_key,
+  task_sequence<f, propertiesT>> : std::true_type {};
+
+template <uint32_t Size>
+inline constexpr invocation_capacity_key::value_t<Size> invocation_capacity;
+
+template <> struct is_property_key<invocation_capacity_key> : std::true_type {};
+
+template <typename ReturnT, typename ... ArgsT, ReturnT(&f) (ArgsT...),
+  class propertiesT>
+struct is_property_key_of<invocation_capacity_key,
+  task_sequence<f, propertiesT>> : std::true_type {};
+
+template <uint32_t Size>
+inline constexpr response_capacity_key::value_t<Size> response_capacity;
+
+template <> struct is_property_key<response_capacity_key> : std::true_type {};
+
+template <typename ReturnT, typename ... ArgsT, ReturnT(&f) (ArgsT...),
+  class propertiesT>
+struct is_property_key_of<response_capacity_key,
+  task_sequence<f, propertiesT>> : std::true_type {};
+
+} // namespace experimental
+} // namespace intel
+} // namespace ext
+} // namespace sycl
+----
+
+--
+[options="header"]
+|===
+| Property | Description
+| balanced | The `balanced` property is a guarantee to the SYCL device compiler
+that the `task_sequence` object will call exactly the same number of `async` s
+and `get` s over the object's lifetime (i.e. before the `task_sequence`
+destructor is invoked). 
+
+| invocation_capacity | The `invocation_capacity` property defines the minimum
+required number `Size` of `async` invocations that must be invokable without any
+`get` being called on that same set.  This number of `async` invocations without
+a `get` call is the minimum number that will be supported before the `async`
+member function blocks.
+
+| response_capacity | The `response_capacity` property defines the maximum
+number `Size` of outstanding `async` invocations such that all outstanding
+invocations are guaranteed to make forward progress. Further `async` invocations
+beyond `Size` may block until enough `get` calls are invoked such that the
+number of outstanding `async` invocations is reduced to `Size`.
+|===
+--
+
+=== Compatibility with FPGA Kernel Interface Properties
+
+The `task_sequence` may be declared with the FPGA Kernel Interface `pipelined`
+property described in link:sycl_ext_oneapi_kernel_properties.asciidoc[
+  sycl_ext_oneapi_kernel_properties].
```

Comment:
This interaction should probably be mentioned in the dependencies section as well.

Reply:
Added.

Reply:
This has not yet been addressed and it's the wrong extension name.

Reply:
This link is still dead.

Reply:
links work now.